src/www/app/server/index.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 The Outline Authors
```

Comment:
The OutlineServer type is an implementation detail of the repository. 
Which makes it also odd that this is the index.ts, since the main part of the library is the repository.
The server should actually be private, only accessible to the module.

If anything, index.ts should be the server repository, and this would be in a server.ts file. I guess the module would have to be renames to something like server_repository

Reply:
Alright, sure!